README.md

```diff
@@ -1,24 +1,139 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# 📗 Table of Contents
 
-Things you may want to cover:
+- [📖 About the Project](#about-project)
+  - [📹 video presentation](#video-presentation)
+  - [:camera: screenshots](#screenshots)
```

Comment:
- [ ] [OPTIONAL] Kindly remove the Video presentation and the Screenshots also from the table of content since you do not have them in the body of the README. This will help maintain a clean code.

Reply:
fixed